config/vufind/searches.ini

```diff
@@ -8,10 +8,11 @@ default_handler      = AllFields    ; Search handler to use if none is specified
 ; below.
 default_sort         = relevance
 
-; This setting controls the sort order to be used for empty search when relevance
-; sort is selected. Since relevance doesn't have a meaningful function with an empty
-; search, this can be set to e.g. "title".
-;empty_search_relevance_override = title
+; This setting controls the sort order to be used for empty (or purely ID-based)
+; searches when relevance sort is selected. Since relevance doesn't have a
+; meaningful function without query terms, this can be set to e.g. "title" or
+; "year desc".
+empty_search_relevance_override = year desc
```

Comment:
I think this (actually, other sorts as well) should contain `, id desc` as a tie-breaker. Otherwise the order of records that have the same year is undetermined.

Reply:
I didn't intend to commit the change from "title" to "year desc" as the default value here -- that was an accident. I've reverted it.

We could certainly change the other example in the comment to "year desc,id desc" if you think it important... though maybe "year desc,title asc" would be more meaningful. I'd think that sorting by ID is just about as arbitrary as sorting by nothing in particular (except that it might survive reindexing with greater consistency).

Reply:
Ok. Let's discuss any changes to the sort fields separately as it's not directly related to the matter at hand.